### PR TITLE
Refactor ContentCardComponentMapper to render correct cards

### PR DESCRIPTION
**Pull request by jordanwade**
## DESCRIPTION
Fixes an issue discovered on The Porch where cards where rendering as the wrong cards. Which made titles truncate and get cut in half.

## Bug
image

## Fix
image

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-ui-connected/src/ContentCardConnected/ContentCardComponentMapper/index.js [hunk 1]
```diff
@@ -20,4 +19,9 @@ ContentCardComponentMapper.defaultProps = {
 
 ContentCardComponentMapper.displayName = 'ListItemCard';
 
-export default ContentCardComponentMapper;
+const ContentCardComponentMapperWithTheme = withTheme(
+  () => ({}),
+  'ui-connected.ContentCardComponentMapper'
```
Comment (conrad-vanl) [inline packages/apollos-ui-connected/src/ContentCardConnected/ContentCardComponentMapper/index.js:24]: So the only real change here is changing this name? From `'ContentCardComponentMapper` to `ui-connected.ContentCardComponentMapper`?

Comment (conrad-vanl) [reply packages/apollos-ui-connected/src/ContentCardConnected/ContentCardComponentMapper/index.js:24]: On The Porch, we were purposefully using a custom component mapper: https://github.com/watermarkchurch/theporch-apollos/blob/master/ThePorch/src/theme/theme.js#L168

I think this PR would break that?

Comment (jordanwade) [reply packages/apollos-ui-connected/src/ContentCardConnected/ContentCardComponentMapper/index.js:24]: Ah, that might be the really issue. I didn't see that custom mapper. So, I might just need to adjust the `PorchFeaturedCard` to get things fixed.

**Comment by jordanwade** [comment]
Fixed in a PR in `The Porch`
